Source/DafnyCore/Resolver/Resolver.cs

```diff
@@ -5818,6 +4612,7 @@ public class ResolveTypeOption {
     /// context. 
     /// </summary>
     public void CheckLocalityUpdates(Statement stmt, ISet<LocalVariable> localsAllowedInUpdates, string where) {
+      // TODO it looks like this method has no side-effects and doesn't return anything.
```

Comment:
Mysterious. We should probably investigate that more deeply at some point, so I'm adding a comment to make it slightly more visible. Given that this is a purely refactoring PR, I don't think we need to include resolving it here, though.